docs/service-catalog/docs/050-gs-broker-registration.md

```diff
@@ -0,0 +1,80 @@
+---
+title: How to register a broker
+type: Getting Started
+---
+
+This Getting Started guide shows how to register a new broker to the Service Catalog. The broker can be either a Namespace-scoped ServiceBroker or a cluster-wide ClusterServiceBroker. Follow the example of the [UPS Broker](https://github.com/kubernetes-incubator/service-catalog/tree/master/charts/ups-broker) to complete the guide.
+
+## Prerequisites
+
+To register a new broker to the Service Catalog, you must have:
```

Comment:
This sentence is redundant, these are prerequisites so... I understand I need these items to proceed :p